### ISSUE TITLE
Alt text?

### ISSUE THREAD

**Issue by codl** [codl/mastodon-apod#109]
Some apod posts have meaningful alt text:

> The featured image shows four planets lined up behind the RFK Triboro bridge in New York City. The image was taken just before sunrise two days ago. Please see the explanation for more detailed information.

<https://apod.nasa.gov/apod/ap220420.html>

Some don't:

> See Explanation.  Clicking on the picture will download the highest resolution version available.

<https://apod.nasa.gov/apod/ap220421.html>

I would like to avoid relaying useless alt text as much as possible, because I think alt text that wastes everyone's time is worse than no alt text, but I would also love for images to be captioned.

One could also imagine a system where the bot owner is asked to describe pictures if the bot isn't confident about the source alt text,  and I personally am OK with hand writing descriptions every morning (it's good practice 😊), but I don't know how I feel about delaying posts indefinitely if I'm not around to describe.

**Comment by codl** [comment]
Here's the alt texts for the last 100 APoD publications https://f.codl.fr/2204/alts.csv?v=1

A quick scan seems to imply that all the useless alt texts are, verbatim,

    See Explanation.  Clicking on the picture will download
    the highest resolution version available.

and the useful ones all end with 

    Please see the explanation for more detailed information.

which could probably be stripped out.

**Comment by codl** [comment]
https://apod.nasa.gov/apod/ap220109.html has

    The featured image shows Jupiter full face including the 
    Great Red Spot as captured by Hubble in 2016.

which does not mention *the explanation* at all
